lms/services/exceptions.py

```diff
@@ -281,6 +263,23 @@ def __init__(
         self.details = details
 
 
+class FileNotFoundInCourse(SerializableError):
+    """A file wasn't found in the current course."""
+
+    def __init__(self, family, file_id):
+
+        ERROR_CODE_MAP = {
+            Family.D2L: "d2l_file_not_found_in_course",
```

Comment:
I like to have the map here instead of doing something like `{family}_file_not_found`

I reckon I'll forget about this in a week, with this approach the error code is more easily grepeable (and a comment would go out of date).